packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -329,9 +329,12 @@ export const deleteFile = async (bucketName: string, filepath: string) => {
   await makeSureBucketExists(objectStore, bucketName)
   const params = {
     Bucket: bucketName,
-    Key: filepath,
+    Key: sanitizeKey(filepath),
   }
-  return objectStore.deleteObject(params)
+  return objectStore.deleteObject(params, function(err, data) {
+    if (err) console.log(err, err.stack); // an error occurred
+    else     console.log(data);           // successful response
```

Comment:
I think we'll want to remove this, logging the delete response isn't necessary here. 

Instead of adding a callback function we should use `objectStore.deleteObject(params).promise()`, perhaps this is why the error wasn't being logged in the exception handler before, is that why you've added this?

Reply:
> deleteObject without callback function cannot delete object successful 

Good catch! I've been able to replicate as well. 

Appending `.promise()` here fixed it for me, which is consistent with how we used the rest of the S3 SDK

Reply:
I have changed to .promise() instead callback for this pull request